src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -35,8 +35,9 @@ def acr_import(cmd,
                target_tags=None,
                resource_group_name=None,
                repository=None,
-               force=False):
-
+               force=False,
+               no_wait=False,
+               ):
```

Comment:
@CarolineNB small style comment: please remove the comma after the last parameter. Also `):` does not need to be on a new line.